modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -124,7 +124,8 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
             as QCLastChangeTime FROM files
             LEFT JOIN files_qcstatus USING (FileID)
             WHERE OutputType='native' AND AcquisitionProtocolID 
-            NOT IN (1, 2, 3, 52) GROUP BY files.SessionID) nd 
+            IN (".
+            implode(",", array_keys($scan_id_types)) .") GROUP BY files.SessionID) nd
```

Comment:
$scan_id_types comes from line [81](https://github.com/gluneau/Loris/blob/774594cd9a8cef1d58d045bf0c1ffa7a400f96c3/modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc#L81) and got its values from the settings "tblScanTypes" at line [65](https://github.com/gluneau/Loris/blob/774594cd9a8cef1d58d045bf0c1ffa7a400f96c3/modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc#L65)

These are the protocols you use, then its the ones that will be re-used here.